docs/performance/inserts/methods.rst

```diff
@@ -123,6 +126,16 @@ For example, using the CrateDB Python client, the following is possible:
 Here, you can vary the number of rows being inserted without having to change
 the prepared statement.
 
+.. WARNING::
+
+    When inserting using ``UNNEST``, (e.g.) using the Python client, CrateDB
```

Comment:
```suggestion
    When inserting using ``UNNEST``, CrateDB
```

will need re-wrapping

Reply:
Or "e.g. when you use..."? Assuming that's what you mean by wrapping

Reply:
I think we can just drop the whole bit about "when using the Python client". this advice applies to any client. at least, that's my understanding! (only caught this when reviewing again today sorry)

Reply:
Alright! It was there in the original so I didn't want to fiddle with it